modules/util/path.go

```diff
@@ -283,38 +283,47 @@ func CommonSkip(name string) bool {
 	return false
 }
 
-// IsReadmeFileName reports whether name looks like a README file
+type FileType string
+
+const (
+	FileTypeReadme  FileType = "readme"
+	FileTypeLicense FileType = "license"
+)
+
+// IsFileName reports whether name looks like a target name file
 // based on its name.
-func IsReadmeFileName(name string) bool {
+func IsFileName(name string, fileType FileType) bool {
 	name = strings.ToLower(name)
-	if len(name) < 6 {
+	lenFileType := len(fileType)
+	if len(name) < lenFileType {
```

Comment:
That's not right, file type doesn't mean file name here. So `len` would cause bugs in the future.

For example:

```
FileTypeCodeOfConduct FileType = "COC"
```

Reply:
35f08001023e6bfdaf081e1957d17c38bb858489